src/runtime/ranger/ranger_resource_policy_manager.cpp

```diff
@@ -36,6 +64,39 @@ void register_rpc_access_type(access_type ac_type,
         ac_type_of_rpc.emplace(code, ac_type);
     }
 }
+
+// Used to map access_type matched resources policies json string.
+std::map<std::string, access_type> access_type_maping({{"READ", access_type::KRead},
+                                                       {"WRITE", access_type::KWrite},
+                                                       {"CREATE", access_type::KCreate},
+                                                       {"DROP", access_type::KDrop},
+                                                       {"LIST", access_type::KList},
+                                                       {"METADATA", access_type::KMetadata},
+                                                       {"CONTROL", access_type::KControl}});
+
+// Parse Ranger ACL policies in JSON format 'data' into 'policies'.
+void parse_policies_from_json(const rapidjson::Value &data, std::vector<policy_item> &policies)
+{
+    CHECK(policies.empty(), "Ranger policy list should not be empty.");
+    RETURN_VOID_IF_NOT_ARRAY(data);
+    for (auto &item : data.GetArray()) {
+        policy_item pi;
```

Comment:
In `struct policy_item`, add the default value `access_type::KInvalid` to access_types like:
```
struct policy_item
{
    access_type access_types = access_type::KInvalid;
    ...
```
Then the assignment here can be omitted.